publish.html

```diff
@@ -629,6 +629,57 @@
 <div class="line"><br /></div> <!-- end #line -->
 </div> <!-- end #line-block -->
 
+<h2><a class="anchor" id="repository"></a>Using OMERO as a Public Repository</h2>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<p>OMERO can be used as a public data repository with either a custom web app or using the OMERO.gallery app.</p>
+
+<p>There are a number of repositories based on OMERO with bespoke front ends, including:</p>
+
+<ul>
+
+<li><p><a href="http://jcb-dataviewer.rupress.org" target="_blank" class="external-url">Journal of Cell Biology (JCB) Data Viewer</a></p></li>
+
+<li><p><a href="http://odr.stowers.org/websimr/" target="_blank" class="external-url">Stowers Institute Original Data Repository (ODR)</a></p></li>
+
+<li><p><a href="http://www.cellimagelibrary.org" target="_blank" class="external-url">
+The Cell Image Library</a></p></li>
+
+</ul>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<p>OMERO.gallery provides a basic gallery front end based on the Group-Project-Dataset-Image, and is a good starting point to develop a bespoke front end.
+
+<p>An example of this can be seen at:
+
+<ul>
+
+<li><p><a href="http://cci02.liv.ac.uk/gallery/" target="_blank" class="external-url">Liverpool  Centre for Cell Imaging (CCI) OMERO Gallery</a></p></li>
+
+</ul>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<p>Full documentation on how to set up OMERO as a public repository and details of customisation options can be found in the <a href="http://www.openmicroscopy.org/site/support/omero5.1/sysadmins/public.html#public-user" target="_blank" class="external-url">OMERO Repository documentation</a>.</p>
```

Comment:
Doc link needs changing to https://www.openmicroscopy.org/site/support/omero5.1/sysadmins/index.html#optimizing-omero-as-a-data-repository